spec/integration/users_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  # before(:each) do
+  #   User.create(user_name: 'Joshep Thiele')
+  # end
```

Comment:
- [ ] Kindly, remove this piece of old commented code in order to improve the repo's cleanliness.  👍